DQM/L1TMonitorClient/python/L1TObjectsTimingClient_cff.py

```diff
@@ -0,0 +1,303 @@
+import FWCore.ParameterSet.Config as cms
+from DQMServices.Core.DQMEDHarvester import DQMEDHarvester
+
+# directory path shortening
+l1tobjectstimingDqmDir = 'L1T/L1TObjects/'
+
+# L1TObjects Timing Ratio Plots
+l1tObjectsRatioPlots = DQMEDHarvester("DQMGenericClient",
+    subDirs = cms.untracked.vstring(l1tobjectstimingDqmDir),
+    efficiency = cms.vstring(
+       "Ratio_L1TMuons_BX_minus2 'Ratio for L1TMuons for BX=-2' L1TMuon/timing/muons_eta_phi_bx_minus2 denominator",
```

Comment:
The paths to the numerator and denominator histograms must be relative to the directory given in subDirs.